themes/bootstrap3/less/components/record.less

```diff
@@ -217,6 +217,12 @@
 .tab-pane .result {
   margin-left: 0;
 }
+.holdings-tab table {
+  table-layout: auto;
+  th:first-child, td:first-child {
+    width: 25%;
+  }
```

Comment:
In response to the discussion on the main page, I think we can remove this part and let the table auto layout. We can do a min-width if we're worried about collapse of like 20%.